src/components/TodoApp.js

```diff
@@ -0,0 +1,194 @@
+    const newMember = members[members.length - 1];
+
+    store.dispatch(createMember(newMember));
+  }
+
+  async addTodo({ code, target }) {
+    if (code !== KEY.ENTER) return;
+    if (!target.classList.contains('new-todo')) return;
+
+    const todoContents = target.value;
+    if (todoContents.length < 2) {
+      alert(MESSAGGE.CREATE_CONTENTS_VALIDATE_ERROR);
+      return;
+    }
+
+    const teamId = getTeamId();
+    const memberId = getMemberId(target);
+    const contents = target.value;
+
+    const todoItem = await todoAPI.createTodoItem(teamId, memberId, { contents });
+    if (!todoItem) return;
+
+    // 상태 업데이트
+    store.dispatch(addTodoItem(memberId, todoItem));
+  }
+
+  async deleteTodo(teamId, memberId, itemId) {
+    await todoAPI.deleteTodoItem(teamId, memberId, itemId);
+
+    store.dispatch(deleteTodoItem(memberId, itemId));
+  }
+
+  async toggleTodo(teamId, memberId, itemId) {
+    const updatedToggleTodoItem = await todoAPI.toggleTodoItem(teamId, memberId, itemId);
+    if (isEmptyObject(updatedToggleTodoItem)) return;
+
+    store.dispatch(toggleTodoItem(memberId, itemId, updatedToggleTodoItem));
+  }
+
+  async updateTodoContents(teamId, memberId, itemId, contents) {
+    const updatedContentsTodoIem = await todoAPI.updateTodoItemContents(teamId, memberId, itemId, {
+      contents,
+    });
+    if (isEmptyObject(updatedContentsTodoIem)) return;
+
+    store.dispatch(updateContentsTodoItem(memberId, itemId, updatedContentsTodoIem));
+  }
+
+  async allDeleteTodo(teamId, memberId) {
+    await todoAPI.allDeleteTodo(teamId, memberId);
+
+    store.dispatch(allDeleteTodoItem(memberId));
+  }
+
+  async changePirority({ target }) {
+    if (!target.classList.contains('chip')) return;
+
+    const teamId = getTeamId();
+    const memberId = getMemberId(target);
+    const itemId = target.closest('li').id;
+
+    const selectValue = target.value;
+    if (selectValue === PRIORITY['select']) return;
+
+    const updatedPriortyTodoItem = await todoAPI.priorityTodoItem(teamId, memberId, itemId, {
+      priority: selectValue,
+    });
+
+    if (isEmptyObject(updatedPriortyTodoItem)) return;
+
+    store.dispatch(changePirortyTodoItem(memberId, itemId, updatedPriortyTodoItem));
+  }
+
+  changeFilter(memberId, target) {
+    const filter = target.dataset['type'];
+
+    store.dispatch(changeFilter(memberId, filter));
+  }
+
+  openEditMode({ target }) {
+    if (target.classList.value !== 'label') return;
+
+    const todoItem = target.closest('li');
+    todoItem.classList.add('editing');
+  }
+
+  closeEditMode({ target, key }) {
+    if (!(key === KEY.ESC || key === KEY.ENTER)) return;
+    if (target.classList.contains('new-todo')) return;
+
+    const todoItem = target.closest('li');
+    if (key === KEY.ESC) {
+      todoItem.classList.remove('editing');
+      return;
+    }
+
+    const teamId = getTeamId();
+    const memberId = getMemberId(target);
+    const itemId = target.id;
```

Comment:
teamId, memberId, itemId를 가져오는 동작이 반복되고 있으니 이 부분을 함수로 분리하면 어떨까요!
```suggestion
    const { teamId, memberId, itemId } = getTargetInfo()??
```
이런 방법 처럼요!!! ㅋㅋㅋ